### DISCUSSION TITLE
NEW VERSION AVAILABLE (3.20)

### DISCUSSION THREAD

**Discussion by biglimp** [UMEP-dev/UMEP#419] [Announcement]
New version out including SOLWEIG 2022a where @nilswallenberg have included anisotrophic longwave radiation scheme. Paper soon available (under review). Other small bug fixes is also included.

**Comment by ML426** [comment]
Hello, 
is there any important modification with the last version on the anisotropic sky.
Because compared to the 2019a version, I observe important differences in terms of mean radiant temperature (Tmrt)
It appears to be more underestimated.
Can you tell me more about this?
Kind regards,

**Reply by biglimp** [reply]
There are many changes made in the longwave part of the model. We have introduced sky patches the same way as we have for shortwave which improves the direction of radiation towards a human as well as a scheme to include anisotrophic longwave sky.  A paper is about to come out very soon. 
What is more underestimated and where, compared to what? Please give more info if you what to suggestions on how to setup the model in the best way possible.

**Reply by ML426** [reply]
I test the sensitivity of the models in terms of Tmrt.
And compared to the measurement, the Tmrt of solweigh is more underestimated, in particular when the area is in the shade (despite the update of the model)

**Reply by biglimp** [reply]
Measurement where? In what kind of shade (buildings or trees)? How did you observe Tmrt? If you used radiometers, please specify which flux (longwave or shortwave) that is underestimated. What forcing data did you use, i.e. how did you derive diffuse and direct shortwave radiation?

**Reply by ML426** [reply]
It is shading from trees.
Tmrt is from radiometers and forcing data from a weather station near the observations site. Diffuse and direct shortwave radiation are estimated from global radiation.
Lup is underestimated but Ldown is overestimated.
Kup and Kdown depend on the time of day, and some hours are overestimated and others underestimated. But during the Peak, they are underestimated

**Reply by biglimp** [reply]
If K fluxes are underestimated you might need to adjust transmissivity of radiation through vegetation. That will also alter Lup.
